components/datatypes/marketing/commerce.schema.json

```diff
@@ -129,6 +129,11 @@
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "Initializations of a new Requisition List."
         },
+        "xdm:commerce.requisitionListDeletes": {
```

Comment:
Didn't catch this yesterday, sorry - should just be "xdm:requisitionListDeletes" - no need for "commerce."